doc/DEMO.md

```diff
@@ -0,0 +1,624 @@
+
+### Python (kinda optional)
+We use python to quickly generate a couple of Bitcoin keys. If you don't have Python on your computer it's fine, but you'll have to find a way to generate Bitcoin keys by yourself.
+
+## Downloading the repositories
+Let's start by creating a dedicated folder - this way if you don't like Revault it will be easier to erase everything :D
+```
+mkdir revault_tutorial
+cd revault_tutorial
+```
+Download all the needed repositories: `cosignerd`, `coordinatord`, `revaultd`:
+```
+git clone https://github.com/revault/coordinatord
+git clone https://github.com/revault/cosignerd
+git clone https://github.com/revault/revaultd
+```
+
+You working directory should look like:
+```
+.
+├── coordinatord
+├── cosignerd
+├── revaultd
+```
+
+## Let's get started!
+### 1. Spinning up Bitcoin Core
+In this tutorial we'll use the regtest network. We'll use a custom data directory inside the `revault_tutorial`.
+```
+mkdir bitcoind_data
+```
+
+Start a Bitcoin Core node using
+```
+bitcoind -regtest -daemon -datadir=$PWD/bitcoind_data
+```
+
+### 2. Starting the coordinator
+As we said, we need just one coordinator running, no matter how many stakeholders/managers there are.
+We'll properly update the coordinator configuration later. For now, let's just retrieve the coordinator's noise key.
+Cd into the coordinator:
+```
+cd coordinatord
+```
+
+Build the project:
+```
+cargo build
+```
+
+Coordinatord needs a Postgre database running, we'll spin it up using docker:
+```
+docker run --rm -d -p 5432:5432 --name postgres-coordinatord -e POSTGRES_PASSWORD=revault -e POSTGRES_USER=revault -e POSTGRES_DB=coordinator_db postgres
+```
+
+Now duplicate the config provided:
+```
+cp contrib/config.toml coordinatord_config.toml
+```
+
+and modify it the `postgres_uri`, it should be `postgresql://revault:revault@localhost:5432/coordinator_db`.
+
+Don't bother with the noise keys now, we'll update them later.
+
+Start the project:
+```
+cargo run -- --conf coordinatord_config.toml
+```
+
+Then please keep note of the noise key printed at startup, we'll need it later.
+
+Now kill the coordinator with `CTRL-C`.
+
+Go back to the parent directory for the next step
+```
+cd ../../
+```
+
+Your notes should now look like:
+```
+stakeholder1: *xpriv* *xpub*
+stakeholder2: *xpriv* *xpub*
+manager1: *xpriv* *xpub*
+coordinator: *noise key*
+```
+
+### 3. Configuring the cosigner
+We'll need one cosigner for each stakeholder.
+Cd into the cosignerd:
+```
+cd cosignerd
+```
+
+Let's start by compiling the project (it may take a while):
+```
+cargo build
+```
+
+#### Stakeholder 1
+First of all, create a directory to store all the cosignerd data:
+```
+mkdir cosignerd_1_data
+```
+
+You can find an [example config](https://github.com/revault/cosignerd/tree/master/contrib/config_regtest.toml) to begin with. Copy it to `./cosigner_1_config.toml`
+```
+cp contrib/config_regtest.toml cosigner_1_config.toml
```

Comment:
https://github.com/revault/cosignerd/tree/master/contrib
I see only a config.toml, not config_regtest.toml